### ISSUE TITLE
Video Classification Task Guide Using Undeclared Variables 

### ISSUE THREAD

**Issue by astern21** [huggingface/transformers#29817]
### System Info

- `transformers` version: 4.32.1
- Platform: macOS-10.16-x86_64-i386-64bit
- Python version: 3.11.4
- Huggingface_hub version: 0.17.2
- Safetensors version: 0.3.2
- Accelerate version: 0.24.1
- Accelerate config:    not found
- PyTorch version (GPU?): 2.0.1 (False)
- Tensorflow version (GPU?): 2.13.0 (False)
- Flax version (CPU?/GPU?/TPU?): 0.7.4 (cpu)
- Jax version: 0.4.16
- JaxLib version: 0.4.16
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

### Who can help?

@amyeroberts

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [X] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

1. Follow the Video Classification example in the HuggingFace docs:
https://huggingface.co/docs/transformers/tasks/video_classification

2. When Reaching the step 

`class_labels = sorted({str(path).split("/")[2] for path in all_video_file_paths})
label2id = {label: i for i, label in enumerate(class_labels)}
id2label = {i: label for label, i in label2id.items()}

print(f"Unique classes: {list(label2id.keys())}.")
`

The variable `all_video_file_paths` is not previously defined in the example docs. I had to write a function to retrieve the video file paths myself, as seen below:

`import os

dataset_path = "UCF101_subset" 

all_video_file_paths = []

for split in ["train", "val", "test"]:
    split_path = os.path.join(dataset_path, split)
    for category in os.listdir(split_path):
        category_path = os.path.join(split_path, category)
        for video_file in os.listdir(category_path):
            video_file_path = os.path.join(category_path, video_file)
            all_video_file_paths.append(video_file_path)`

3. When preparing the dataset for training, the variable `dataset_root_path` is not previously defined. 

### Expected behavior

I assume that these variables should have been previously defined to not cause errors for people following the example docs.

**Comment by NielsRogge** [comment]
Thanks for reporting, would you be able to open a PR to fix this?

**Comment by astern21** [comment]
Sure, I'd be happy to help.

**Comment by manju-rangam** [comment]
Hi @amyeroberts , the main version of video classification notebook [https://github.com/huggingface/notebooks/blob/main/examples/video_classification.ipynb] seems to define the all_video_file_paths. Should this issue be considered fixed?

**Comment by amyeroberts** [comment]
Hi @manju-rangam - the issue is referencing the examples in the docs, which should still be updated.

**Comment by manju-rangam** [comment]
@amyeroberts, I've created a PR related to this

**Comment by anirudh2781998** [comment]
Hey @amyeroberts new to open source community can i be considered to work on this issue if its not fixed?

**Comment by NielsRogge** [comment]
The PR above has been merged, so will close this issue. @anirudh2781998 feel free to take a look at other issues labeled with "good first issue" or "good first documentation issue"
